### PR TITLE
Search

### PR DESCRIPTION
**Pull request by jonathanarde**
fix issue https://github.com/TheoTech/glorifycc/issues/30

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -281,15 +281,45 @@ router.get('/search', function(req, res, next) {
     var langFilter = req.body.langFilter
     var totalSongsDisplayed = req.body.totalSongsDisplayed
     var songs2d = []
-    var findOriginalSong = function(done) {
+
+    function findSongsLoggedIn(done) {
```
Comment (daxkerchner) [inline routes/index.js]: Why'd you change to a function declaration here?  AFAIK it's usually better to use function expressions

File: routes/index.js [hunk 2]
```diff
@@ -312,41 +342,24 @@ router.get('/search', function(req, res, next) {
...
-                    Playlist.find({
-                        owner: req.user._id,
-                        song: {
-                            $exists: false
-                        }
-                    }, function(err, playlists) {
-                        if (err) next(err)
-                        res.render('search', {
-                            songs: songs2d,
-                            inLibrary: user.library,
```
Comment (daxkerchner) [inline routes/index.js]: To clarify, are we removing functionality here?

File: routes/index.js (hunk 1 above)
Comment (jonathanarde) [inline routes/index.js]: because i started to like function declaration because of the hoisting behaviour

File: routes/index.js (hunk 2 above)
Comment (daxkerchner) [inline routes/index.js]: Is there core elsewhere to display `user.library` and `playlists` though?

**Pull request merged**